### PR TITLE
Bugfix/baseline links

### PR DESCRIPTION
**Pull request by christopherwharrop-noaa**
## Proposed changes

This PR fixes a bug in the cmake configuration that prevented the baseline test output from being linked properly.

## Types of changes

- [x] Bugfix (Change which fixes an issue)
- [ ] New feature (Change which adds functionality)
- [ ] Enhancement to existing functionality (non-breaking change which improves existing code or documentation)
- [ ] Breaking change (fix or feature that changes results and requires updates to test baselines)
- [ ] Documentation Update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/NOAA-GSL/SENA-c_sw/blob/develop/CONTRIBUTING.md) document
- [x] I have ensured all my changes contribute to a common purpose
- [x] I have verified my change does not add debug code
- [x] I have verified my change does not add code that is commented out
- [x] I have verified unit tests pass locally with my changes
- [x] I have verified my changes adhere to style guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by christopherwharrop-noaa** [comment]
I don't know what was going on before.  I closed the previous PR and opened a new one, and now the test is passing ok.  Please review this one and let me know if you see issues.

File: ref/test/CMakeLists.txt [hunk 1]
```diff
@@ -62,11 +51,12 @@ foreach(FILENAME ${c_sw_test_input})
            ${CMAKE_CURRENT_BINARY_DIR}/${FILENAME} )
 endforeach(FILENAME)
 
+# Create test_output directory for test output and symlink all files
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/test_output)
-foreach(BASELINE_FILENAME TEST_OUTPUT_FILENAME IN ZIP_LISTS c_sw_baselines c_sw_test_output)
+foreach(FILENAME ${c_sw_test_output})
```
Comment (robgpita) [inline ref/test/CMakeLists.txt:56]: This is a much cleaner way to structure the comparison.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
   macos_Build:
     name: Mac OS Build
     # The type of runner that the job will run on
-    runs-on: macos-latest
+    runs-on: macos-10.15
```
Comment (robgpita) [inline .github/workflows/ci.yml:17]: I'd say this is an appropriate fix to get the Github Actions to run the workflow. We don't necessarily need to be running on the latest MacOS (for now).

**Review by robgpita — APPROVED** [review]
Looks good. Thanks for making the code more simple and elegant in ref/test/CMakeLists.txt.

**Pull request merged**